### ISSUE TITLE
Image Gallery Not linkable

### ISSUE THREAD

**Issue by snowite0** [WordPress/gutenberg#12690]
If you have a question you have a few places that you can ask this:

- Support Forums: https://wordpress.org/support/plugin/gutenberg
- Handbook: https://wordpress.org/gutenberg/handbook
- https://chat.wordpress.org #core-editor

If you are unable to ask in those places you can ask here, however you will get faster responses through those recommended places.

**Comment by snowite0** [comment]
I have the most updated version of the new editor. Here is the problem I am experiencing.  I did a layout for a 6 image gallery.  The options for the gallery are: Attachment page, Media or None.  I wanted  to make  the image link able where I want it to go: Ie a page I designate. However, the only link editing option that shows up is for the text on the page.  The entire image is not select able and leave only a text link which messes up the image. See attached screen shots.  I used the first image dog wrapped in lights. see the difference in the two before and after.
![sniglett_net_1](https://user-images.githubusercontent.com/30342297/49679271-23d49300-fa58-11e8-935f-0aca43f0216e.png)
![2 with link](https://user-images.githubusercontent.com/30342297/49679280-318a1880-fa58-11e8-8371-b39da5a59097.png)
